PancakeSwap-Bots/Large-LP-Deposit-Withdrawal/package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "pancakeswap-3",
+  "version": "0.0.1",
+  "description": "Detects large LP deposits/withdrawals to/from a Pancakeswap pool",
+  "repository": "https://github.com/NethermindEth/Forta-Agents/tree/main/PancakeSwap-Bots/Large-LP-Deposit-Withdrawal",
+  "chainIds": [
+    56
+  ],
+  "scripts": {
+    "build": "tsc",
+    "start": "npm run start:dev",
+    "start:dev": "nodemon --watch src --watch forta.config.json -e js,ts,json  --exec \"npm run build && forta-agent run\"",
+    "start:prod": "forta-agent run --prod",
+    "tx": "npm run build && forta-agent run --tx",
+    "block": "npm run build && forta-agent run --block",
+    "range": "npm run build && forta-agent run --range",
+    "file": "npm run build && forta-agent run --file",
+    "publish": "forta-agent publish",
+    "push": "forta-agent push",
+    "disable": "forta-agent disable",
+    "enable": "forta-agent enable",
+    "keyfile": "forta-agent keyfile",
+    "test": "jest --detectOpenHandles",
+    "format": "prettier --write \"src/**/*.ts\""
+  },
+  "dependencies": {
+    "forta-agent": "^0.1.8",
+    "lru-cache": "^7.10.2"
+  },
+  "devDependencies": {
+    "@types/jest": "^27.0.1",
+    "@types/nodemon": "^1.19.0",
+    "forta-agent-tools": "^2.2.1",
```

```suggestion
    "forta-agent-tools": "^2.2.2",
```

Please 1) delete package-lock.json 2) change this to `^2.2.2` 3) run `npm install` and 4) push